{
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unknown",
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "In the DataBuf class in include/exiv2/types.hpp in Exiv2 0.26, an issue exists in the constructor with an initial buffer size. A large size value may lead to a SIGABRT during an attempt at memory allocation. NOTE: some third parties have been unable to reproduce the SIGABRT when using the 4-DataBuf-abort-1 PoC file."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-02-11T00:00:00",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "https://bugzilla.novell.com/show_bug.cgi?id=1087879",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://bugzilla.novell.com/show_bug.cgi?id=1087879"
                },
                {
                    "name": "https://github.com/xiaoqx/pocs/tree/master/exiv2",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://github.com/xiaoqx/pocs/tree/master/exiv2"
                },
                {
                    "name": "GLSA-201811-14",
                    "tags": [
                        "x_refsource_GENTOO"
                    ],
                    "url": "https://security.gentoo.org/glsa/201811-14"
                },
                {
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1564281",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1564281"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2018-9145",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "In the DataBuf class in include/exiv2/types.hpp in Exiv2 0.26, an issue exists in the constructor with an initial buffer size. A large size value may lead to a SIGABRT during an attempt at memory allocation. NOTE: some third parties have been unable to reproduce the SIGABRT when using the 4-DataBuf-abort-1 PoC file."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.novell.com/show_bug.cgi?id=1087879",
                            "refsource": "MISC",
                            "url": "https://bugzilla.novell.com/show_bug.cgi?id=1087879"
                        },
                        {
                            "name": "https://github.com/xiaoqx/pocs/tree/master/exiv2",
                            "refsource": "MISC",
                            "url": "https://github.com/xiaoqx/pocs/tree/master/exiv2"
                        },
                        {
                            "name": "GLSA-201811-14",
                            "refsource": "GENTOO",
                            "url": "https://security.gentoo.org/glsa/201811-14"
                        },
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1564281",
                            "refsource": "MISC",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1564281"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2018-9145",
        "dateUpdated": "2022-02-11T00:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}